package.json

```diff
@@ -6,150 +6,153 @@
+    "babel-loader": "8.0.0-beta.4",
     "babel-plugin-styled-components": "^1.5.1",
-    "browser-sync": "~2.23.6",
-    "connect-history-api-fallback": "~1.5.0",
-    "css-loader": "~0.28.11",
-    "ejs": "~2.5.8",
-    "enzyme": "~3.3.0",
-    "enzyme-adapter-react-16": "~1.1.1",
-    "enzyme-to-json": "^3.3.3",
-    "escape-string-regexp": "~1.0.5",
-    "eslint": "~4.19.1",
-    "eslint-config-standard": "~11.0.0",
-    "eslint-config-standard-react": "~6.0.0",
+    "browser-sync": "^2.24.5",
+    "connect-history-api-fallback": "^1.5.0",
+    "css-loader": "^1.0.0",
+    "ejs": "^2.6.1",
+    "enzyme": "^3.3.0",
+    "enzyme-adapter-react-16": "^1.1.1",
+    "enzyme-to-json": "^3.3.4",
+    "escape-string-regexp": "^1.0.5",
+    "eslint": "^5.1.0",
+    "eslint-config-standard": "^11.0.0",
+    "eslint-config-standard-react": "^6.0.0",
     "eslint-plugin-flowtype": "^2.50.0",
-    "eslint-plugin-jest": "~21.15.0",
-    "eslint-plugin-prefer-object-spread": "~1.2.1",
-    "eslint-plugin-react": "~7.7.0",
-    "eslint-plugin-standard": "~3.0.1",
-    "file-loader": "~1.1.11",
-    "flow-bin": "^0.69.0",
-    "front-matter": "~2.3.0",
+    "eslint-plugin-jest": "^21.17.0",
+    "eslint-plugin-prefer-object-spread": "^1.2.1",
+    "eslint-plugin-react": "^7.10.0",
+    "eslint-plugin-standard": "^3.1.0",
+    "file-loader": "^1.1.11",
+    "flow-bin": "^0.76.0",
+    "flow-typed": "^2.5.1",
+    "front-matter": "^2.3.0",
     "generate-package-json-webpack-plugin": "^1.0.0",
-    "git-repo-version": "~1.0.2",
-    "highlight.js": "~9.12.0",
-    "html-loader": "~0.5.5",
-    "identity-obj-proxy": "~3.0.0",
-    "img-loader": "~2.0.1",
+    "git-repo-version": "^1.0.2",
+    "highlight.js": "^9.12.0",
+    "html-loader": "^0.5.5",
+    "identity-obj-proxy": "^3.0.0",
+    "imagemin": "^5.3.1",
+    "img-loader": "^3.0.0",
     "is-url": "^1.2.4",
-    "jest": "~22.4.3",
-    "jest-fetch-mock": "~1.5.0",
-    "json-loader": "~0.5.7",
-    "lolex": "~2.3.2",
-    "path-to-regexp": "~2.2.0",
-    "pixrem": "~4.0.1",
-    "pleeease-filters": "~4.0.0",
-    "postcss": "~6.0.21",
-    "postcss-calc": "~6.0.1",
-    "postcss-color-function": "~4.0.1",
-    "postcss-custom-media": "~6.0.0",
-    "postcss-custom-properties": "~7.0.0",
-    "postcss-custom-selectors": "~4.0.1",
-    "postcss-flexbugs-fixes": "~3.3.0",
-    "postcss-import": "~11.1.0",
-    "postcss-loader": "~2.1.3",
-    "postcss-media-minmax": "~3.0.0",
-    "postcss-mixins": "~6.2.0",
-    "postcss-nesting": "~4.2.1",
-    "postcss-selector-matches": "~3.0.1",
-    "postcss-selector-not": "~3.0.1",
-    "prettier-eslint": "^8.8.1",
-    "prop-types": "~15.6.1",
-    "raf": "~3.4.0",
-    "react-hot-loader": "~4.0.1",
-    "react-scroll": "~1.7.8",
-    "redux-mock-store": "~1.5.1",
-    "regenerator-runtime": "^0.11.1",
-    "rimraf": "~2.6.2",
-    "s3": "~4.4.0",
-    "style-loader": "~0.20.3",
-    "stylefmt": "~6.0.0",
-    "stylelint": "~9.2.0",
+    "jest": "22.4.4",
```

Comment:
same here

Reply:
Because we need version 22 because of an incompatibility with the IDE